lib/bls.ex

```diff
@@ -77,4 +81,15 @@ defmodule Bls do
       {:error, _} -> false
     end
   end
+
+  @doc """
+  Converts private to public key
+  """
+  @spec derive_pubkey?(privkey()) :: {:ok, pubkey()} | {:error, any()}
+  def derive_pubkey?(private_key) do
+    case Bls.derive_pubkey(private_key) do
+      {:ok, pubkey} -> pubkey
+      {:error, _} -> false
+    end
```

Comment:
We should do something similar to [`Ssz.hash_tree_root!`](https://github.com/lambdaclass/lambda_ethereum_consensus/blob/main/lib%2Fssz.ex#L51-L55) here: match against `{:ok, pk}`, and return `pk`. That way, if the Rust code returns an error, we just raise it.